lib/screens/explore_screens/flag/flag_user/interactor/usecases/flag_user_usecase.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:seeds/datasource/local/settings_storage.dart';
+import 'package:seeds/datasource/remote/api/flag_repository.dart';
+import 'package:seeds/datasource/remote/model/transaction_response.dart';
+import 'package:seeds/domain-shared/base_use_case.dart';
+
+class FlagUserUseCase extends InputUseCase<TransactionResponse, String> {
+  final FlagRepository _flagRepository = FlagRepository();
+
```

Comment:
Usually we create these on the fly, any reason it's stored in a variable? 

Not a bug, just style question.

Reply:
nope, fixed